pkgs/by-name/po/poptracker/package.nix

```
@@ -26,14 +29,15 @@ stdenv.mkDerivation (finalAttrs: {
   patches = [ ./assets-path.diff ];
 
   postPatch = ''
-     substituteInPlace src/poptracker.cpp --replace "@assets@" "$out/share/$pname/"
+    substituteInPlace src/poptracker.cpp --replace "@assets@" "$out/share/$pname/"
```

We should be consistent, either `${finalAttrs.pname}` or `$pname` or string literal.
I don't see much benefit from using `finalAttrs` here personally, I'd rather just use the literal...

---

Yes, `pname` shouldn't be reused somewhere else actually. Please use a string literal.
```suggestion
    substituteInPlace src/poptracker.cpp --replace "@assets@" "$out/share/poptracker/"
```

---

Fixed